src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -136,7 +136,9 @@ public InstrumenterResult Instrument()
 
         private void InstrumentModule()
         {
-            using (var stream = new FileStream(_module, FileMode.Open, FileAccess.ReadWrite))
+            var fileSystem = (IFileSystem)DependencyInjection.Current.GetService(typeof(IFileSystem));
```

Comment:
We shoud avoid `DependencyInjection` reference in code...we need to inject throught contructor and update al callers.

Reply:
Done